app/components/provider_relationship_permissions_list.html.erb

```diff
@@ -1,11 +1,29 @@
+<p class="govuk-!-margin-bottom-1 govuk-body">The following organisation(s) can make decisions:</p>
 <ul class="govuk-list">
-  <li>View applications <% @permission_model.view_applications_only? ? 'only' : nil %></li>
-
-  <% if @permission_model.make_decisions? %>
-    <li><%= render CheckIcon.new %> Make decisions</li>
+  <% [:training_provider, :ratifying_provider].each do |provider| %>
+    <% if @permission_model.send("#{provider}_can_make_decisions?".to_sym) %>
+      <li><%= render CheckIcon.new %> <%= @permission_model.send(provider).name %> </li>
+    <% end %>
   <% end %>
+</ul>
 
-  <% if @permission_model.view_safeguarding_information? %>
-    <li><%= render CheckIcon.new %> View safeguarding information</li>
+<p class="govuk-!-margin-bottom-1 govuk-body">The following organisation(s) can see safeguarding information:</p>
+<ul class="govuk-list">
+  <% [:training_provider, :ratifying_provider].each do |provider| %>
```

Comment:
You could put a method on `ProviderRelationshipPermissionsList`

```ruby
def providers_that_can(permission)
  providers_that_can = ['training_provider', 'ratifying_provider'] map do |provider_role|
    if @permission_model.send("#{provider_role}_can_#{permission}")
      @permission_model.send(provider_role)
    end
  end

  providers_that_can.compact
end
```

And we could define `(training|ratifying)_provider_can_view_applications_only?` on `ProviderRelationshipPermissions`.

The structs aren't that useful here as it's all so meta. I think we should prob get rid of them for now?